wks-recruiter/src/main/java/pl/lodz/p/it/wks/wksrecruiter/services/PositionServiceImpl.java

```diff
@@ -25,4 +26,14 @@ public Position addPosition(Position position) throws WKSRecruiterException {
             throw new WKSRecruiterException(new WKSRecruiterException.Error("NAME_NOT_UNIQUE", "Position with such name already exists. Try another one."));
         }
     }
+
+    @Override
+    public boolean modifyPosition(String name, boolean value) throws WKSRecruiterException {
+        Position position = positionsRepository.findPositionByName(name);
+        if (position == null) {
+            throw new WKSRecruiterException(new WKSRecruiterException.Error("POSITION_NOT_FOUND", "Position with such name does not exist."));
```

Comment:
czemu new od new?

Reply:
Bo Error to klasa wewnętrzna w WKSRecruiterException, który zawiera listę errorów